package.json

```diff
@@ -23,11 +26,18 @@
   },
   "license": "MIT",
   "dependencies": {
+    "asset-pipe-common": "^1.0.0-beta.6",
     "browser-pack": "6.0.3",
-    "get-stream": "^3.0.0"
+    "browserify": "^15.2.0",
+    "deep-equal": "^1.0.1",
```

Comment:
https://www.npmjs.com/package/fast-deep-equal

Reply:
doesn't like this is being used anyway

Reply:
Even better